### PR TITLE
Tooling update v2

### PR DESCRIPTION
**Pull request by nowyDEV**
# What

- Added test for accordion component
- Updated eslint config
- Added additional eslint plugins
- Fixed eslint/ts/stylelint errors
- Converted some rules to warnings to fix them in a future
- Publish only `dist` folder

## Compatibility

- [ ] Does this change maintain backward compatibility?

### CODE REVIEW

**Review by mrMarshalX — APPROVED** [review]

File: src/components/input/InputUpload.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { FunctionComponent } from "react";
+import React from "react";
```
Comment (mrMarshalX) [inline src/components/input/InputUpload.tsx:1]:
```suggestion
import * as React from "react";
```

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nordcloud/gnui",
   "description": "Nordcloud Design System - a collection of reusable React components used in Nordcloud's SaaS products",
-  "version": "2.8.0",
+  "version": "3.0.0",
```
Comment (Bosmanfrx) [inline package.json:4]: Are there any breaking changes? Or publishing only `dist` directory is considered one?

File: src/components/dropdown/Dropdown.tsx [hunk 3]
```diff
@@ -73,7 +73,7 @@ export const Dropdown: FunctionComponent<DropdownProps & SpaceProps> = ({
       <DropdownButton
         name={name}
         onClick={() => !disabled && setIsOpen(!isOpen)}
-        disabled={disabled && disabled}
```
Comment (Bosmanfrx) [inline src/components/dropdown/Dropdown.tsx:76]: 🤣

File: src/components/pagetitle/Pagetitle.tsx [hunk 4]
```diff
@@ -13,7 +12,6 @@ type PageTitleProps = {
 type PageTitleBreadcrumbsProps = {
   title: string;
   list: Array<BreadcrumbsList>;
-  margin?: string;
```
Comment (Bosmanfrx) [inline src/components/pagetitle/Pagetitle.tsx:16]: I think it needs to be removed from story as well

**Review by mrMarshalX — APPROVED** [review]

**Pull request merged**